### PR TITLE
fix: Bump kapeta deps and adjust types

### PR DESCRIPTION
**Pull request by andertun**

### CODE REVIEW

**Comment by andertun** [comment]
npm publish

**Comment by andertun** [comment]
npm publish

**Comment by andertun** [comment]
npm publish

File: package.json [hunk 1]
```diff
@@ -32,20 +32,20 @@
         "test": "echo \"Error: no tests\""
     },
     "peerDependencies": {
-        "@kapeta/ui-web-types": "<2",
-        "@kapeta/ui-web-utils": "<2",
+        "@kapeta/ui-web-types": "^1.3.1",
+        "@kapeta/ui-web-utils": "^0.0.29",
```
Comment (wejendorp) [inline package.json:36]: Not sure we want to narrow these?

**Review by wejendorp — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (andertun) [reply package.json]: Ok, changed it to `<2` and `<1` respectively

Comment (wejendorp) [reply package.json]: I'd say leave them as open as we can, while ensuring compatibility. i.e. `>=0.0.29 < 1` is fine, if `0.0.29` has something we need. But to optimize the dependency tree, open is good :)

**Pull request merged**